en/parse.adoc

```diff
@@ -944,17 +944,23 @@ parse [remove me <and me also> "but leave me be"][some [remove word!] mark: to s
 
 ==== `insert`
 
-Inserts literal value or result of expression evaluation at the current position. Always succeeds and advances the input past the insertion.
+Inserts literal value or result of expression evaluation, either at the current position or at the marked one. Always succeeds. Advances the input past the insertion if it was made at the current position, otherwise input position is retained.
```

Comment:
What is the "marked" position referred to here?

Reply:
@greggirwin see [section 3.5.1](https://github.com/red/docs/blob/master/en/parse.adoc#marking).